### PR TITLE
Add parsing or expression

### PR DESCRIPTION
**Pull request by mareklabonarski**
Parsing '|' was not implemented. Now, expression like
((((EAaMemLteDspPoolCategory_Shared) << (16))) | (0)) 
is properly parsed and calculated.

### CODE REVIEW

**Comment by aurzenligl** [comment]
I'm a bit confused, #30 is not delivered yet and this PR (#31) seems to include content of previous one.

**Comment by mareklabonarski** [comment]
I needed one reference for both of changes, so I branched out #31 out of #30, although they are not related...

**Comment by mareklabonarski** [comment]
this is ready. Can we pull it in?

**Comment by aurzenligl** [comment]
We'll pull that, but let's pull #30 first. I've pulled docstring fix with `requirements.txt` change -> which results in conflict in both #30 and #31. Would you rebase #30 first, so that we can pull it, and then we'll pull this one?

**Comment by aurzenligl** [comment]
(
I allowed myself to rebase on current master to keep history linear
`git remote add mareklabonarski git@github.com:mareklabonarski/prophy.git`
`git fetch --all`
`git rebase -i origin/master --committer-date-is-author-date`
`git push -f`
)

**Review by aurzenligl — APPROVED** [review]
LGTM (although adding a unit test for `|` would be great)

**Comment by aurzenligl** [comment]
@mareklabonarski, would you like to have 1.2.3 released once this PR is merged, or do you have some further changes?

**Comment by mareklabonarski** [comment]
No further changes as for now, thanks. Let's make 1.2.3...

**Comment by aurzenligl** [comment]
Released: https://pypi.org/project/prophy/1.2.3/

**Comment by mareklabonarski** [comment]
Dzieki!

czw., 6 cze 2019, 00:35 użytkownik Krzysztof Laskowski <
notifications@github.com> napisał:

**Pull request merged**
